packages/core/integration-tests/test/server.js

```diff
@@ -452,6 +452,11 @@ describe('server', function() {
         name: 'index.html',
         assets: ['index.html'],
       },
+      {
+        name: 'index.HASH_REF_2d285d6f29236f5b00e1db199488d505.css',
```

Comment:
This hash isn't necessarily stable, it should work correctly even without the `name:` property.

Reply:
Perfect, I pushed the change. Probably there is a minor bug in assertBundles, because inline bundles have to be specified before index.html otherwise, assertBudles fails.